### PR TITLE
add new article on adding vpn users

### LINKED ISSUE #1558: New article - Add VPN user in my.rackspace portal
Trevor has written a new article on adding a VPN user in the my.rackspace portal. He says it's been approved by his management to be customer-facing. The article can be found in the wiki, and the link is in the email from Alan on 10/21.

### PR DESCRIPTION
**Pull request by stephamon**
Fixes #1558

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a9abe66257/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e72a1215a4/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-7b3f3e4ff4/how-to/)

File: content/dedicated-hosting/add-new-vpn-user-myrackspace-portal.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### Limitations
+
+VPN user automation currently does not allow for the following modifications:
+
+- Custom passwords
+- VPN-filters
+- Group-lock
+- Framed IP addresses
+- Remove a VPN user
```
Comment (cyrichardson) [inline content/dedicated-hosting/add-new-vpn-user-myrackspace-portal.md]: VPN user removal

File: content/dedicated-hosting/add-new-vpn-user-myrackspace-portal.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Modify a VPN user password
```
Comment (cyrichardson) [inline content/dedicated-hosting/add-new-vpn-user-myrackspace-portal.md]: VPN user password modification

**Comment by cyrichardson** [comment]
@stephamon Just a couple of suggestions (to keep all nouns in the bulleted list). Let me know what you think.

**Comment by cyrichardson** [comment]
Ooops sorry hit the wrong button. Reopened.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-47af1d6cbd/how-to/)

**Comment by cyrichardson** [comment]
LGTM - merged

**Pull request merged**